README.md

```diff
@@ -186,11 +195,22 @@ All the microservices communicate with the Postgres Container based on the confi
 
 ![Launch Portal](LaunchPortal.jpg)
 
-Upon clicking 'Login from Google' and providing Google Credentials(if the same Google Account is used where the developer credentials came from), the system redirects the user to their landing page as shown below:
+Upon clicking 'Login from Google' and providing Google Credentials (if the same Google Account is used where the developer credentials came from), the system redirects the user to their landing page as shown below:
 
 
 ![Logged Into Portal](LoggedInScreenshot.jpg)
 
+
+### Revproxy-service cannot start
+If revproxy-service cannot start an error will occur. It may be useful to 
+```
+docker-compose down
+docker-compose up -d
+```
+If the error still occurs, make sure that apache2 and revproxy-service do not share the same port. You can change the port for revproxy-service and any other service in the `docker-compose.yaml` [file](https://github.com/uc-cdis/compose-services/blob/master/docker-compose.yml#L215). For revproxy you would also need to change the port in the `nginx.conf` [here](https://github.com/uc-cdis/compose-services/blob/master/nginx.conf#L29).
```

Comment:
when linking to a line in a code or config file that may change, in general it's good to link to a commit instead of master, so the links don't die or become misleading